src/__tests__/resources/timeSeriesObject.unit.spec.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 Cognite AS
+import MockAdapter from 'axios-mock-adapter';
+import {
+  CogniteClient,
+  DatapointsPostDatapoint,
+  PostTimeSeriesMetadataDTO,
+} from '../../index';
+import { Asset } from '../../resources/classes/asset';
+import { TimeSeriesList } from '../../resources/classes/timeSeriesList';
+import { randomInt, setupLoggedInClient } from '../testUtils';
+
+describe('TimeSeries class unit test', () => {
+  let axiosMock: MockAdapter;
+  let client: CogniteClient;
+  let newTimeSeries: PostTimeSeriesMetadataDTO;
+  let createdTimeSeries: TimeSeriesList;
+  let timeSeriesWithAssetId: PostTimeSeriesMetadataDTO;
+  let datapointArray: DatapointsPostDatapoint[] = [];
+  beforeAll(() => {
+    client = setupLoggedInClient();
+    axiosMock = new MockAdapter(client.instance);
+  });
+  beforeEach(async () => {
+    axiosMock.reset();
+
+    // Create timeseries
+    newTimeSeries = {
+      name: 'test-timeseries',
+      externalId: 'timeseries' + randomInt(),
+    };
+    timeSeriesWithAssetId = {
+      ...newTimeSeries,
+      assetId: randomInt(),
+      externalId: 'timeseriesWithAssetId' + randomInt(),
+    };
+    axiosMock
+      .onPost(new RegExp('/timeseries$'), {
+        items: [newTimeSeries, timeSeriesWithAssetId],
+      })
+      .replyOnce(200, {
+        items: [newTimeSeries, timeSeriesWithAssetId].map(timeseries => ({
+          ...timeseries,
+          id: randomInt(),
+        })),
+      });
+    createdTimeSeries = await client.timeseries.create([
+      newTimeSeries,
+      timeSeriesWithAssetId,
+    ]);
+
+    // Create datapoints
+    datapointArray = [];
```

Comment:
Same here. Why not create the `datapointArray` manually or with a for-loop? I Don't see any benefit of mocking the datapoints.insert -request